### PR TITLE
https://issues.redhat.com/browse/ACM-9681 known issue with Kube/API

### PR DESCRIPTION
**Pull request by swopebe**
https://issues.redhat.com/browse/ACM-9681

### CODE REVIEW

File: release_notes/known_issues_observability.adoc [hunk 1]
```diff
@@ -26,6 +26,33 @@ Review the known issues for {product-title}. The following list contains known i
...
+----
+
+Thanos receives pods logs with the following errors:
+
+----
+caller=handler.go:551 level=error component=receive component=receive-handler tenant=xxxx err="no matching hashring to handle tenant" msg="internal server error"
+----
+
+See the following procedure to resolve this issue:
+
```
Comment (swopebe) [inline release_notes/known_issues_observability.adoc:51]: these are confusing to me, we are telling the user to go through 1 step to 0 and one back to 2 or N? In that order? Or are these two different methods?

Comment (subbarao-meduri) [reply release_notes/known_issues_observability.adoc:51]: @swopebe
The deployment name is `observability-observatorium-api`. By default, they are deployed as 2 instances. However, customer could have changed number of instances to more than 2 if scaling is needed. The fix is
1. Scale down  number of instances from N -> 0 
2. Scale up number of instances from 0 -> N 
(N = 2 by default, but could be > 2 in some environments, if custom configuration is used)

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

File: release_notes/known_issues_observability.adoc [hunk 2]
```diff
@@ -26,6 +26,35 @@ Review the known issues for {product-title}. The following list contains known i
 For your {ocp} cluster, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/release_notes/ocp-4-12-release-notes#ocp-4-12-known-issues. 
 
 For more about deprecations and removals, see xref:../release_notes/deprecate_remove.adoc#deprecations-removals[Deprecations and removals].
+[#permission-to-managed-cluster-monitoring-denied]
+
+== Observatorium API gateway pods in a restored hub cluster might have stale tenant data
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc]:
```suggestion

[#permission-to-managed-cluster-monitoring-denied]
== Observatorium API gateway pods in a restored hub cluster might have stale tenant data
```

File: release_notes/known_issues_observability.adoc [hunk 3]
```diff
@@ -26,6 +26,35 @@ Review the known issues for {product-title}. The following list contains known i
...
+//2.9:9681
+
+The Observatorium API gateway pods in a restored hub cluster might contain stale tenant data after a backup and restore procedure because of a Kubernetes limitation. See link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically[Mounted ConfigMaps are updated automatically] for more about the limitation.
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc:34]: I see that "ConfigMap" is used. If I am recalling correctly, I believe that is why we had instances of `ConfigMap`. We wanted to be consistent with Kubernetes. With seeing this, we should be aligned with Kubernetes. Let's discuss with the team

File: release_notes/known_issues_observability.adoc [hunk 4]
```diff
@@ -26,6 +26,35 @@ Review the known issues for {product-title}. The following list contains known i
...
+
+As a result, the Observatorium API and Thanos gateway rejects metrics from collectors, and the {product-title-short} Grafana dashboards do not display data.
+
+See the following errors from the Observatorium API gateway pod logs:
+
+----
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc:40]:
```suggestion
[source,bash]
----
```

File: release_notes/known_issues_observability.adoc [hunk 5]
```diff
@@ -26,6 +26,35 @@ Review the known issues for {product-title}. The following list contains known i
...
+level=error name=observatorium caller=logchannel.go:129 msg="failed to forward metrics" returncode="500 Internal Server Error" response="no matching hashring to handle tenant\n"
+----
+
+Thanos receives pods logs with the following errors:
+
+----
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc:46]:
```suggestion
[source,bash]
----
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_observability.adoc (hunk 3 above)
Comment (swopebe) [reply release_notes/known_issues_observability.adoc:34]: @dockerymick  ConfigMap can be used, but what we don't want is configmap or configMap or anything else we try here.

Comment (swopebe) [reply release_notes/known_issues_observability.adoc:34]: Here is the guidance: yes[ config map (noun)](https://redhat-documentation.github.io/supplementary-style-guide/#config-map)
Description: In Red Hat OpenShift, a config map holds configuration data for pods to consume. The API object for a config map is ConfigMap.

So you are right, we need to code it or write it out.

File: release_notes/known_issues_observability.adoc (hunk 4 above)
Comment (swopebe) [inline release_notes/known_issues_observability.adoc:40]: Not so sure about this one, we actually decided not to do this but I see it here and there in the doc now; I think some folks started doing on on their own. I think this is valid for a bash command. This is an error, however. I am going to reject this and bring it up again tomorrow.

File: release_notes/known_issues_observability.adoc (hunk 5 above)
Comment (swopebe) [inline release_notes/known_issues_observability.adoc:46]: same, it is an error and not a bash command. (???) Interested in the reason for _source, bash_ here, but we have not discussed using this across the team. Let's chat about it tomorrow.

File: release_notes/known_issues_observability.adoc (hunk 3 above)
Comment (swopebe) [reply release_notes/known_issues_observability.adoc:34]: @dockerymick I am going to leave the title the way it is here, but normally, yes we would code. Here is why:

This is the exact title of the third part documentation, it isn't our content.

File: release_notes/known_issues_observability.adoc (hunk 5 above)
Comment (dockerymick) [reply release_notes/known_issues_observability.adoc:46]: My reason is because it is displayed from the terminal. I considered CLI views as "bash"

**Review by dockerymick — APPROVED** [review]

File: release_notes/known_issues_observability.adoc (hunk 5 above)
Comment (swopebe) [reply release_notes/known_issues_observability.adoc:46]: Ok, I understand, but want to see what others use for errors. These would potentially build with a copy block but and error would not need that.

**Pull request merged**